### PR TITLE
Cleanup cache calls

### PR DESCRIPTION
**Pull request by kasper-piskorski**

### CODE REVIEW

**Comment by kasper-piskorski** [comment]
!rtg

**Comment by kasper-piskorski** [comment]
!rtg - timeout

**Comment by kasper-piskorski** [comment]
!rtg - com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: localhost/127.0.0.1:9042 (com.datastax.driver.core.exceptions.OperationTimedOutException:

**Comment by kasper-piskorski** [comment]
!rtg

File: grakn-graql/src/main/java/ai/grakn/graql/internal/reasoner/atom/binary/RelationshipAtom.java [hunk 1]
```diff
@@ -93,8 +93,10 @@
 
     private int hashCode = 0;
     private Multimap<Role, Var> roleVarMap = null;
+    private Multimap<Role, SchemaConcept> roleTypeMap = null;
```
Comment (fppt) [inline grakn-graql/src/main/java/ai/grakn/graql/internal/reasoner/atom/binary/RelationshipAtom.java]: `roleMap` but meh you can fix it in another PR

File: grakn-graql/src/main/java/ai/grakn/graql/internal/reasoner/atom/binary/RelationshipAtom.java [hunk 2]
```diff
@@ -324,14 +325,16 @@ public int computePriority(Set<Var> subbedVars) {
     }
 
     private Multimap<Role, SchemaConcept> getRoleTypeMap() {
```
Comment (fppt) [inline grakn-graql/src/main/java/ai/grakn/graql/internal/reasoner/atom/binary/RelationshipAtom.java]: `getRoleMap`. . .  but again, meh

**Pull request merged**